vendors/vendor/boards/board/aws_demos/application_code/main.c

```diff
@@ -186,47 +195,44 @@ void vApplicationDaemonTaskStartupHook( void )
 {
     /* FIX ME: Perform any hardware initialization, that require the RTOS to be
      * running, here. */
-    
 
-    /* FIX ME: If your MCU is using Wi-Fi, delete surrounding compiler directives to 
-     * enable the unit tests and after MQTT, Bufferpool, and Secure Sockets libraries 
-     * have been imported into the project. If you are not using Wi-Fi, see the 
+
+    /* FIX ME: If your MCU is using Wi-Fi, delete surrounding compiler directives to
```

Comment:
Is this comment accurate? For one thing, we have ports that use FreeRTOS+TCP with Wi-Fi. For another, I think we have ports with offloaded TCP/IP that call vApplicationIPNetworkEventHook as part of their integration with FreeRTOS.